.github/workflows/forge-ci.yml

```
@@ -118,88 +102,70 @@ jobs:
-            })
+          path: ExocoreGateway.base.json
+          key: ExocoreGateway-base-layout-${{ github.event.pull_request.head.sha || github.event.after || github.sha }}
+      - name: Generate storage layout files for the PR
+        run: |
+          for file in Bootstrap ClientChainGateway RewardVault Vault ExocoreGateway ExoCapsule; do
+            forge inspect src/core/${file}.sol:${file} storage-layout > ${file}.compiled.json;
+          done
+      - name: Copy back base branch layout file
+        run: cp /tmp/ExocoreGateway.base.json ./ExocoreGateway.base.json
+      - name: Zip storage layout files
+        run: zip storage-layouts.zip *.compiled.json ExocoreGateway.base.json
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2035:info:1:25: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)